lib/logproto/logproto-buffered-server.c

```diff
@@ -142,10 +142,21 @@ log_proto_buffered_server_convert_from_raw(LogProtoBufferedServer *self, const g
                 }
               break;
             case EILSEQ:
-            default:
-              msg_notice("Invalid byte sequence or other error while converting input, skipping character",
+              msg_notice("Invalid byte sequence, skipping character",
                          evt_tag_str("encoding", self->super.options->encoding),
                          evt_tag_printf("char", "0x%02x", *(guchar *) raw_buffer));
+              raw_buffer++;
+              avail_in--;
+
+              state->pending_buffer_end = state->buffer_size - avail_out;
+              *((gchar *)(self->buffer + state->pending_buffer_end)) = '\xFF';
```

Comment:
0xFF will cause problems everywhere on the pipeline, I think we should just skip the invalid character.

If we don't want to silently drop invalid byte sequences, I suggest returning additional information to LogReader about the incomplete conversion so that it can inject a tag, name-value pair, or a valid marker to the affected `LogMessage`.

Reply:
Reverted back to the original solution. Feel free to push to this PR any time :)

Reply:
Sorry, I didn't mean to add an 0xFF character, rather a sequence of characters injected that contains the original offeding character in hexadecimal form.